### PR TITLE
Updates for 0.11

### PR DESCRIPTION
**Pull request by jacereda**

### CODE REVIEW

File: test/Main.purs [hunk 1]
```diff
@@ -5,7 +5,7 @@ module Test.Main where
 
   import Optic.Core
 
-  import Prelude (class Show, Unit, (#), ($), (+), (<>), bind, const, show)
+  import Prelude (class Show, Unit, (#), ($), (+), (<>), const, show, discard)
```
Comment (joneshf) [inline test/Main.purs]: I'm not up on what all happened, why is this change necessary?

**Review by joneshf — COMMENTED** [review]
Thanks for this! I have one question for you, but otherwise good to merge.

File: test/Main.purs (hunk 1 above)
Comment (jacereda) [inline test/Main.purs]: Fixes this error:

    Unknown value discard. You're probably using do-notation, which the compiler replaces with calls to the discard function. Please import discard from module Prelude

See `Implicitly discarded values in do blocks now raise errors` in https://github.com/purescript/purescript/releases/tag/v0.11.0

**Comment by paf31** [comment]
@joneshf Can this be merged now? Thanks.
